content/docs/user-guide/experiment-management/cleaning-experiments.md

```diff
@@ -78,7 +78,7 @@ $ dvc exp show --all-branches
 
 ```dvctable
 ┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
-┃ white:**Experiment**    ┃ white:**Created**      ┃    yellow:**acc** ┃ blue:**model.conv_units** ┃
+┃ neutral:**Experiment**              ┃ neutral:**Created**      ┃    metric:**acc** ┃ param:**model.conv_units** ┃
 ┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
```

Comment:
Checked in https://dvc-org-ref-dvctable-ojpnqwjsg.herokuapp.com/doc/user-guide/experiment-management/cleaning-experiments.

![image](https://user-images.githubusercontent.com/1477535/131789816-250aeb13-50fc-4ddb-b0c0-69e3123387f0.png)